qucs-doc/report/EKV26/Makefile.am

```diff
@@ -29,6 +29,8 @@ EXTRA_DIST = $(SOURCES)
 SOURCES = EKV26.tex content.tex
 TARGETS = EKV26.pdf
 
+export TEXINPUTS=$(srcdir):$(srcdir)/..:
```

Comment:
> less cluttered in this way

no. global variables are *more* cluttered. seriously.

>  every invocation of pdflatex

the number of invocations wll reduce drastically with 'include'.

> Why exporting TEXINPUTS here is bad?

exporting is bad. what happens here is (semantically) argument passing. it's not your fault that latex does not support `-I`. but that's what it is. look at how a compiler is invoked: `$(CC) $(CCFLAGS)`, every time, over and over again.

argument passing is a *local* thing, so it should happen in the very same line that runs latex.

(hope that's more clear now...?)